pkg/chains/signing.go

```diff
@@ -252,10 +280,30 @@ func (ts *TaskRunSigner) SignTaskRun(ctx context.Context, tr *v1beta1.TaskRun) e
 			}
 		}
 		if merr.ErrorOrNil() != nil {
+			ok, s := IsRetrying(tr)
```

Comment:
Let's move all of this into a separate function so the current function doesn't get too long

Reply:
Moved it to a separate function.